tests/testthat/test-basics.R

```diff
@@ -357,4 +369,16 @@ test_that("NAs aren't dropped in strings2factor() (#1291)", {
     bake(new_data = NULL)
 
   expect_identical(rec_res, ex_data)
-})
\ No newline at end of file
+})
+
+test_that("recipe() can handle very long formulas (#1283)", {
```

Comment:
this test is simply here to demonstrate that we doesn't get a `Error: C stack usage  7954280 is too close to the limit` error

Reply:
Ha, that's a lot of columns. Hell yeah!